FAQ.md

```diff
@@ -0,0 +1,85 @@
+
+## Basic
+### I have a cable service and I want those channels to be on this IPTV. Is that possible?
+No. This playlist does not replace traditional cable, because this playlist can't include exclusive channels that may not be available on free-to-air. We recommend contacting your cable provider to offer legal alternatives to this service or visit the [iptv-org/awesome-iptv](https://github.com/iptv-org/awesome-iptv) repository for find other providers.
+
+### Does the playlist have a channel guide?
+Yes. However, not all channels have their EPG available. See [EPG](https://github.com/iptv-org/epg) for more info. Please note that the guide is not perfect and some channels (like international feeds) may interrupt programming without you knowing.
+
+### Is possible to view channels directly from the main website?
+No, to avoid possible abuse of the service.
+
+### Why don't we show NSFW channels in most playlists?
+Because to avoid content that may be inappropriate for work or display to the entire audience. We suggest using the corresponding playlist.
+
+### Do I have to pay to use this list?
+No. Also, the maintenance of this service is self-financed.
+
+### Are you planning to include a VOD of TV shows broadcasted on this playlist?
+No. This is not our purpose.
+
+## Channels
+### Does the channel need to be publicly accessible to be included in the playlists?
+Yes, we only need the links to be publicly accessible like FTA packages (free-to-air), online services or official websites. We recommend tagging [Geo-blocked] if it is only accessible in one or few countries.
+
+### Can I add a FTA channel that broadcasts PPV events or pay programming?
+Maybe. Consider if the country where it is broadcasted allows fair use. Owners can ban the broadcast of PPV events, and even DCMA takedowns. Check the blocklist from the repository database for more info about specific events.
+
+### I wish created my channel and add it to this playlist. What should I do?
+We are a community initiative and you don't need to pay anything to stay on this playlist. Feel free to create a channel with few resources:
+* Camera with a tripod (recommended for your own studio).
+* A PC with Internet, video editor and recording/broadcasting app under RTMP protocol (e.g. OBS).
+* And a live streaming service with multibitrate (if possible).
+
+After you have your m3u8 link, add your channel information from [Database](https://github.com/iptv-org/database).
+
+## Technical
+### Why is recommended the "master.m3u8", "playlist.m3u8" or "index.m3u8" at the end of the url?
+These files are useful to see perfectly the channel based in your network or screen (adaptive resolution). Also the streams are permanent, without token, that avoids edit temporally their urls.
+
+### Why don't we accept XStreamCode in streams?
+XStreamCode contains code that could be malicious to users.
```

Comment:
```suggestion
Xtream-Codes streams are, most of the times, pirated streams created by some people that can get satellite and/or PPV streams and that propose it to everyone for a fairer price. These types of servers are illegal and also highly unstable, since it depends either from the user who bought it and the server that hosts it.
```